lighthouse-core/config/config.js

```diff
@@ -438,6 +442,28 @@ class Config {
     return merge(baseJSON, extendJSON);
   }
 
+  /**
+   * @param {LH.Config.Json} configJSON
+   * @param {string=} configDir
+   * @return {LH.Config.Json}
+   */
+  static mergePlugins(configJSON, configDir) {
+    const pluginNames = configJSON.plugins;
+
+    if (pluginNames) {
+      for (const pluginName of pluginNames) {
+        const filename = `lighthouse-plugin-${pluginName}`;
```

Comment:
~~does this mean you can't pass in a path to a plugin? it seems like that might make it a bit easier to develop plugins locally.~~

I see in the tests the hack you can use to get this to work with non-npm modules, but looks like it would still have to be colocated with the config file or in the cwd?

Reply:
Cool cool makes sense